Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -766,88 +771,503 @@ def add_user_to_channel_command():
+def delete_team_request(team_id: str) -> None:
+    url = f'{GRAPH_BASE_URL}/v1.0/groups/{team_id}'
+    http_request('DELETE', url)
+
+
+def delete_team():
+    team_id = demisto.args().get('team_id')
+    delete_team_request(team_id)
+    demisto.results(f'Team {team_id} was deleted successfully.')
+
+
+@logger
+def list_members_request(team_id: str):
+    url = f'{GRAPH_BASE_URL}/v1.0/teams/{team_id}/members'
+    return http_request('GET', url)
+
+
+def list_members():
+    team_id = demisto.args().get('team_id')
+    response = list_members_request(team_id)
+    members = response.get('value', [])
+    return_results(CommandResults(
+        outputs_prefix='MicrosoftTeams.TeamMember',
+        outputs_key_field='id',
+        readable_output=tableToMarkdown(
+            f'Team {team_id} Members List',
+            members,
+            ['id', 'displayName', 'email', 'roles'],
+        ),
+        outputs=members,
+        raw_response=response,
+    ))
+
+
+@logger
+def get_member_request(team_id: str, membership_id: str):
+    url = f'{GRAPH_BASE_URL}/v1.0/teams/{team_id}/members/{membership_id}'
+    return http_request('GET', url)
+
+
+def get_member():
+    team_id = demisto.args().get('team_id')
+    membership_id = demisto.args().get('membership_id')
+    team_member = get_member_request(team_id, membership_id)
+    return_results(CommandResults(
+        outputs_prefix='MicrosoftTeams.TeamMember',
+        outputs_key_field='id',
+        readable_output=tableToMarkdown(
+            f'Team Member {membership_id} Details',
+            team_member,
+            ['id', 'displayName', 'email', 'roles'],
+        ),
+        outputs=team_member,
+        raw_response=team_member,
+    ))
+
+
+@logger
+def add_member_request(team_id: str, user_id: str, roles: Optional[list] = None) -> dict:
+    url = f'{GRAPH_BASE_URL}/v1.0/teams/{team_id}/members'
+    return cast(Dict[Any, Any], http_request(
+        'POST',
+        url,
+        json_={
+            '@odata.type': '#microsoft.graph.aadUserConversationMember',
+            'roles': roles,
+            'user@odata.bind': f"https://graph.microsoft.com/v1.0/users('{user_id}')",
+        }
+    ))
+
+
+def add_member():
+    team_id = demisto.args().get('team_id', '')
+    user_id = demisto.args().get('user_id', '')
+    team_member = add_member_request(
+        team_id=team_id,
+        user_id=user_id,
+        roles=['owner'] if argToBoolean(demisto.args().get('is_owner', 'false')) else [],
+    )
+    return_results(CommandResults(
+        outputs_prefix='MicrosoftTeams.TeamMember',
+        outputs_key_field='id',
+        readable_output=tableToMarkdown(
+            f'User {user_id} was added to the team {team_id} successfully.',
+            team_member,
+            ['id', 'displayName', 'email', 'roles'],
+        ),
+        outputs=team_member,
+        raw_response=team_member,
+    ))
+
+
+@logger
+def remove_member__request(team_id: str, membership_id: str) -> None:
```

Comment:
i'm guessing it's by mistake
```suggestion
def remove_member_request(team_id: str, membership_id: str) -> None:
```